astropy/wcs/wcs.py

```diff
@@ -84,7 +84,7 @@
            'NoWcsKeywordsFoundError', 'InvalidTabularParametersError']
 
 
-if six.PY3 or platform.system() == 'Windows':
+if (not six.PY2) or platform.system() == 'Windows':
```

Comment:
No (as per the previous comment, I have no strong opinions, but it looks clearer to me, at a first glance)